packages/fx-core/tests/component/generator/generator.test.ts

```diff
@@ -38,6 +38,7 @@ import templateConfig from "../../../src/common/templates-config.json";
 import { placeholderDelimiters } from "../../../src/component/generator/constant";
 import sampleConfigV3 from "../../common/samples-config-v3.json";
 import Mustache from "mustache";
+import * as folder from "../../../../fx-core/src/folder";
```

Comment:
This import is a little confusing. I prefer using `import * as folderUtils from ...`. You can make the decision by yourself.

Reply:
Thanks for the feedback. It's true that it's less confusing to write that way.